spec/fixtures/parature_faqs/filter_date.json

```diff
@@ -1,16 +1,13 @@
 {
-  "filter": {
+  "query": {
     "bool": {
       "must": [
         {
-          "range": {
-            "update_date": {
-              "from": "2013-01-11",
-              "to": "2013-03-18"
-            }
+          "query_string": {
+            "query": "update_date:[2013-01-11 TO 2013-03-18]"
```

Comment:
I'm curious as to if/why the "query_string" approach is better than the "range" approach. Any thoughts on that @tmhammer?